### ISSUE TITLE
Phone Number returns empty string

### ISSUE THREAD

**Issue by rupamhazra** [pbakondy/cordova-plugin-sim#89]
### Expected behaviour
Tell us what should happen.

### Actual behaviour
Tell us what happens instead. Provide a log message if relevant

### I'm seeing this behaviour on
_Remove this hint: these checkboxes can be checked like this: [x]_

- [ ] Android device
- [ ] Android emulator

### I am using

- [ ] cordova
- [ ] ionic

#### Hardware models
Example: Samsung Galaxy S6, iPhone 6s,Redmi

#### OS versions
Example: Android 4.4.2, iOS 9.2

### I've checked these
- [ ] It happens on a fresh Cordova CLI project as well.
- [ ] I'm waiting for `deviceready` to fire.
- [ ] My JavaScript has no errors (`window.onerror` catches nothing).
- [ ] I'm using the latest cordova library, Android SDK, Xcode, etc.

### So how can we reproduce this?
Provide the used components versions (cordova, ionic, etc).

Provide the steps to reproduce the issue.

Provide files, sources if available.

**Comment by Sampath-Lokuge** [comment]
Yes, it returns an empty string. Have you found out any solution here?

**Comment by ravargash** [comment]
Has someone from the team has checked on this?
